recipes/general/recipe-cosmic-silverblue-nvidia-userns.yml

```diff
@@ -0,0 +1,19 @@
+name: cosmic-silverblue-nvidia-userns-hardened
+
+description: "Cosmic-silverblue-nvidia with some hardening applied"
+
+base-image: ghcr.io/ublue-os/cosmic-silverblue-nvidia
```

Comment:
do these include gnome? I don't see the need to include these as well if so

Reply:
They do, I just wanted to include them for the time being if users want to try it out alongside another DE, the ublue kinoite variant is borderline unusable

Reply:
let's only include base. as a general rule, only images that we intend to maintain long term should be added to secureblue. Otherwise, we will inevitably have to include notices for deprecated images, like what happened with the framework images

Reply:
Done.